### PR TITLE
PR to test the reviews

### PR DESCRIPTION
**Pull request by ftkghost**
PR to test the reviews.
* file one - Atlassian values.
* file two - Atlassian migration plan.

### CODE REVIEW

**Comment by ftkghost** [comment]
add a comment

File: first-file.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Open company,no bullshit
+Openness is root level for us. Information is open internally by default and sharing is a first principle. And we understand that speaking your mind requires equal parts brains (what to say), thoughtfulness (when to say it), and caring (how it’s said).
```
Comment (ftkghost) [inline first-file.txt:2]: Comment 1 in review 1

File: second-file.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
+The Atlassian Migration Program is here!
+
+The Atlassian Migration Program is here to help migrating customers, from Cloud assessment to adoption. It’s free and available to all (no sign up required). To get started, take the migration assessment for a personalized recommendation on when to migrate and actionable next steps.
+
+Change is hard. But the good news? Migrating to Cloud doesn’t have to be. There’s no better time to start exploring Cloud, because it’s never been easier to do so. We’re launching the Atlassian Migration Program to provide the tools, support, and guidance to help customers confidently make the move.
```
Comment (ftkghost) [inline second-file.txt:5]: Comment 2 in review 1

**Review by ftkghost — COMMENTED** [review]
Add a review 1

File: second-file.txt (hunk 2 above)
Comment (nzsalt) [reply second-file.txt:5]: review

**Comment by nzsalt** [comment]
reviewing

**Pull request merged**